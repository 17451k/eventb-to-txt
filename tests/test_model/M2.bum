<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<org.eventb.core.machineFile org.eventb.core.configuration="org.eventb.core.fwd" version="5">
<org.eventb.core.event name="." org.eventb.core.convergence="0" org.eventb.core.extended="false" org.eventb.core.label="INITIALISATION"/>
</org.eventb.core.machineFile>
