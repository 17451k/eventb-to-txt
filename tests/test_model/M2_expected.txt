machine M2

events
    event INITIALISATION
    end

end
